build/cite/ogcapi-features10/README.md

```diff
@@ -0,0 +1,41 @@
+# OGC API Features - Conformance Test Suite execution
+
+These are the instructions on how to run the CITE Conformance Test Suite for the
+*GeoServer* OGC API Features service.
+
+## Goals
+
+* To run them manually on demand in your local machine
+* To run them while you're debugging GeoServer in your IDE
```

Comment:
Looking at the makefile and other changes.. I was actually wondering how to do this part.
Everything seems to be starting up a docker compose with both geoserver and teamengine, so how does one execute GeoServer inside the IDE while still running a test suite?

Reply:
right, I'm gonna come to it while updating the dev docs

Reply:
Updated the devs guide's `doc/en/developer/source/cite-test-guide/index.rst` this readme accordingly